### PR TITLE
Virtual table to list all the keywords

### PR DESCRIPTION
**Pull request by nirbhayc**
This would also deprecate 'GET [NOT] RESERVED KW'.

### CODE REVIEW

**Comment by riverszhang89** [comment]
YES -> Y; NO -> N.

File: sqlite/ext/comdb2/keywords.c [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+ */
+
+#if (!defined(SQLITE_CORE) || defined(SQLITE_BUILDING_FOR_COMDB2)) &&          \
+    !defined(SQLITE_OMIT_VIRTUALTABLE)
+
+#if defined(SQLITE_BUILDING_FOR_COMDB2) && !defined(SQLITE_CORE)
+#define SQLITE_CORE 1
+#endif
+
+#include <assert.h>
```
Comment (ldennis) [inline sqlite/ext/comdb2/keywords.c]: This file actually broke build with internal builddb in bbenv. Consider adding `#include <string.h>` for `memset`?

Comment (nirbhayc) [inline sqlite/ext/comdb2/keywords.c]: @ldennis I'll try to fix this today.

Comment (nirbhayc) [inline sqlite/ext/comdb2/keywords.c]: https://github.com/bloomberg/comdb2/pull/232

**Pull request merged**